models/wishlist.py

```diff
@@ -45,3 +45,10 @@ class WishListItem(db.Model):
 
     def __repr__(self):
         return '<Wish List Item {}>'.format(self.title)
+
+    @classmethod
+    def deleteWish(cls, wish_id):
+        row = WishListItem.query\
```

Comment:
Are you sure you want it like this ?
why not just `get` ? e.g.:
`row = WishListItem.query.get(wish_id)`

Reply:
I'm also not a fan of generic names like`row`, if possible, please name vars accordingly.